core/sync.js

```diff
@@ -60,8 +60,10 @@ class Sync {
   local: Local
   pouch: Pouch
   remote: Remote
-  stopped: ?boolean
   moveTo: ?string
+  stopRequested: Promise
+  started: ?Promise
+  running: ?Promise<boolean>
```

Comment:
It looks like the code is checking the `started` promise as a boolean, but not the `running` promise:

```js
started: ?Promise<boolean>
running: ?Promise
```